account_balance_line/i18n/es.po

```diff
@@ -20,12 +21,12 @@ msgstr ""
 #. module: account_balance_line
 #: view:account.move.line:account_balance_line.view_account_move_line_filter_balance
 msgid "Account"
-msgstr ""
+msgstr "Cuenta"
 
 #. module: account_balance_line
 #: field:account.move.line,line_balance:0
```

Comment:
This translation is not valid for version 9. Please export from v9 and translate again.